src/main/java/core/basesyntax/strategy/operation/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+
+public class ReturnOperation implements OperationHandler {
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public void operationType(String fruit, int amount) {
+        int newQuantity = storageDao.getFruit(fruit).getQuantity() + amount;
```

Comment:
There's a potential issue if `storageDao.getFruit(fruit)` returns `null`. This would cause a `NullPointerException` when calling `getQuantity()`. Ensure that the `getFruit` method handles the case when the fruit is not found in the storage.